### PR TITLE
feat: add flexbox

### PR DESCRIPTION
**Pull request by Istaak**
## Description

[Add flexbox to project](https://leviahack.atlassian.net/browse/RB-4)

## Solution

Add [react-styled-flexboxgrid](https://github.com/LoicMahieu/react-styled-flexboxgrid#react-styled-flexboxgrid)

## Evidence

<details>
<summary>SS</summary>

image
image

</details>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,13 +24,16 @@
...
+    "prop-types": "^15.0.0-0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-i18next": "^11.14.0",
     "react-redux": "^7.2.0",
     "react-router-dom": "6",
     "react-scripts": "4.0.3",
+    "react-styled-flexboxgrid": "^3.2.1",
     "react-test-renderer": "^17.0.2",
+    "styled-components": "2",
```
Comment (alegabo93) [inline package.json]: I'm going to approve the PR but I'm curious if the `styled-components` package is needed (not a big deal but I believe that emotion already added it).

**Review by alegabo93 — APPROVED** [review]
Overall looks great!

File: package.json (hunk 1 above)
Comment (alegabo93) [reply package.json]: Nevermind. I'll have to add it for the GlobalStyles.

Comment (Istaak) [reply package.json]: haha yes, and also for the installation, they mention to add them XD
image

**Review by JonathanRamirezJ — APPROVED** [review]

**Comment by JonathanRamirezJ** [comment]
Cool!! 😄

**Pull request merged**